core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -5859,18 +5858,17 @@ private void checkLiteral2(String expression, String expected) {
 
   @Test public void testRegexSubstrFunction5ArgswithBackSlash() {
     final String query = "select regexp_substr('chocolate Chip cookies','[-\\_] V[0-9]+',"
-        + " 1,1,'i')\n"
+        + "1,1,'i')\n"
         + "from \"foodmart\".\"product\" where \"product_id\" in (1, 2, 3, 4)";
     final String expected = "SELECT "
-        + "REGEXP_EXTRACT_ALL(SUBSTR('chocolate Chip cookies', 1), '(?i)[-\\\\_] V[0-9]+') [OFFSET(0)]\n"
+        + "REGEXP_SUBSTR('chocolate Chip cookies', '(?i)[-\\\\_] V[0-9]+', 1, 1)\n"
         + "FROM foodmart.product\n"
         + "WHERE product_id = 1 OR product_id = 2 OR product_id = 3 OR product_id = 4";
     sql(query)
         .withBigQuery()
         .ok(expected);
   }
```

Comment:
check whether you can pass string arguments in 3rd/4th parameter

Reply:
Actually 3rd and 4th parameters are of numeric data type at both dialects so test case is failing for same if we pass.